src/server/builds/node.js

```diff
@@ -210,7 +210,7 @@ AlgoliaSearchNodeJS.prototype._request = function request(rawUrl, opts) {
 
       abort();
       clearTimeout(timeoutId);
-      reject(new errors.NodeNetwork(err.message, err));
+      reject(new errors.Network(err.message, err));
```

Comment:
Isn't Network the error sent by the client when there is a CORS error? It might be a bit misleading if we have errors that are bags for several symptoms...